spark/pom.xml

```diff
@@ -34,13 +34,13 @@
   <description>Zeppelin spark support</description>
   <url>http://zeppelin.incubator.apache.org</url>
 
-  <properties>
-    <spark.version>1.4.1</spark.version>
-    <scala.version>2.10.4</scala.version>
-    <scala.binary.version>2.10</scala.binary.version>
-  </properties>
-
-
+  <repositories>
+    <repository>
+      <id>cloudera</id>
+      <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
```

Comment:
Why do we need non-vendor neutral repository here?
It has been rised before, blocking 0.5.0 release, please reffer https://issues.apache.org/jira/browse/ZEPPELIN-404